### PR TITLE
Add new RemoveMilestone method on IssuesService

### LINKED ISSUE #236: IssuesService.Edit cannot clear an issue's milestone
Much like #181 I fail to see how to use go-github to clear an issue's milestone. The API requires the milestone be to set to `null` in the request so that it is cleared, but that is not possible since the field uses `omitempty`.

Not sure what the right solution would be, perhaps a standalone method for clearing milestones?

### PR DESCRIPTION
**Pull request by AGMETEOR**
Fixes #236

### CODE REVIEW

**Comment by AGMETEOR** [comment]
Hi @gmlewis and @cpanato . Mind taking a look at my PR?

File: github/issues.go
Comment (gmlewis) [inline github/issues.go:304]:
```suggestion
	u := fmt.Sprintf("repos/%v/%v/issues/%v", owner, repo, issueNumber)
```

File: github/issues.go
Comment (gmlewis) [inline github/issues.go:306]:
```suggestion
		Milestone *Milestone `json:"milestone"`
```

File: github/issues.go
Comment (gmlewis) [inline github/issues.go:302]:
```suggestion
// Remove a milestone from an issue.
//
// This is a helper method to explicitly update an issue with a `null` milestone, thereby removing it.
//
// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/issues/#update-an-issue
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @AGMETEOR !
Just a couple minor tweaks, please, then we'll be ready for a second LGTM before merging.

**Review by cpanato — COMMENTED** [review]
i'm not a maintainer of this project, but Glenn already did a great job reviewing this pr

besides that LGTM after the comments being addressed 

thanks for working on this!

**Comment by AGMETEOR** [comment]
Hi @gmlewis . I have made updates. Kindly requesting for another review.

**Review by gmlewis — CHANGES_REQUESTED** [review]

File: github/issues.go
Comment (AGMETEOR) [reply github/issues.go]: Committed the suggestion

**Review by gmlewis — APPROVED** [review]
Thank you, @AGMETEOR !
LGTM.

Merging based upon @cpanato additional approval. 
Thank you, @cpanato !

**Pull request merged**
